packages/quoting-api-svc/src/service.ts

```diff
@@ -0,0 +1,391 @@
+
+ Contributors
+ --------------
+ This is the official list (alphabetical ordering) of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Arg Software
+ - Jose Francisco Antunes <jfantunes@arg.software>
+ - Rui Rocha <rui.rocha@arg.software>
+
+ --------------
+ ******/
+
+"use strict";
+
+import {
+	AuditClient,
+	KafkaAuditClientDispatcher,
+	LocalAuditClientCryptoProvider,
+} from "@mojaloop/auditing-bc-client-lib";
+import {AuthenticatedHttpRequester, AuthorizationClient, TokenHelper} from "@mojaloop/security-bc-client-lib";
+import {DefaultConfigProvider, IConfigProvider} from "@mojaloop/platform-configuration-bc-client-lib";
+import {IAuthorizationClient, ITokenHelper} from "@mojaloop/security-bc-public-types-lib";
+import {ILogger, LogLevel} from "@mojaloop/logging-bc-public-types-lib";
+import {MLKafkaJsonConsumer, MLKafkaJsonProducerOptions} from "@mojaloop/platform-shared-lib-nodejs-kafka-client-lib";
+import express, {Express} from "express";
+
+import {GetQuotingConfigSet} from "@mojaloop/quoting-bc-config-lib";
+import {IAuditClient} from "@mojaloop/auditing-bc-public-types-lib";
+import {IConfigurationClient} from "@mojaloop/platform-configuration-bc-public-types-lib";
+import {IMetrics} from "@mojaloop/platform-shared-lib-observability-types-lib";
+import {IBulkQuoteRepo, IQuoteRepo} from "@mojaloop/quoting-bc-domain-lib";
+import {KafkaLogger} from "@mojaloop/logging-bc-client-lib";
+import {MongoBulkQuotesRepo, MongoQuotesRepo} from "@mojaloop/quoting-bc-implementations-lib";
+import {PrometheusMetrics} from "@mojaloop/platform-shared-lib-observability-client-lib";
+import {Server} from "net";
+import {QuotingAdminExpressRoutes} from "./routes/quote_admin_routes";
+import {QuotingPrivilegesDefinition} from "@mojaloop/quoting-bc-domain-lib";
+import {existsSync} from "fs";
+import process from "process";
+import crypto from "crypto";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJSON = require("../package.json");
+
+const BC_NAME = "quoting-bc";
+const APP_NAME = "quoting-api-svc";
+const APP_VERSION = packageJSON.version;
+
+const PRODUCTION_MODE = process.env["PRODUCTION_MODE"] || false;
+const LOG_LEVEL: LogLevel = (process.env["LOG_LEVEL"] as LogLevel) || LogLevel.DEBUG;
+
+const KAFKA_URL = process.env["KAFKA_URL"] || "localhost:9092";
+const MONGO_URL = process.env["MONGO_URL"] || "mongodb://root:mongoDbPas42@localhost:27017/";
```

Comment:
Comment from sonarcloud scan: "Make sure this MongoDB database password gets changed and removed from the code."

Please address this security issue and lets make the best use of the analysis from the available tooling. Analysis on this PR here: https://sonarcloud.io/summary/new_code?id=mojaloop_quoting-bc&pullRequest=17